tools/lsvast/CMakeLists.txt

```diff
@@ -8,4 +8,12 @@ endif ()
 
 add_executable(lsvast lsvast.cpp)
 target_link_libraries(lsvast PRIVATE vast::libvast vast::internal)
+
+target_compile_features(lsvast PUBLIC cxx_std_17)
+
+# GNU prior to 9.1 requires linking with stdc++fs when using std::filesystem library
+if (CMAKE_CXX_COMPILER_ID MATCHES "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 9.1.0)
+  target_link_libraries(lsvast PUBLIC stdc++fs)
+endif()
```

Comment:
Not needed since you linked this publicly against libvast already, which lsvast links against.